src/js/utils/log.js

```diff
@@ -8,6 +8,9 @@ import {isObject} from './obj';
 
 let log;
 
+// This is the private tracking variable for logging level.
+let level = 'debug';
```

Comment:
I guess I'm not 100% sure on this one either. I wonder if it makes more sense to turn logging off and make it opt-in for 6.0?